cellxgene_schema_cli/cellxgene_schema/ontology_files/owl_info.yml

```
@@ -1,23 +1,23 @@
 CL:
-  latest: 2023-06-22
+  latest: 2023-08-24
   urls:
-    2023-06-22: https://github.com/obophenotype/cell-ontology/releases/download/v2023-06-22/cl.owl
+    2023-08-24: https://github.com/obophenotype/cell-ontology/releases/download/v2023-08-24/cl.owl
 EFO:
-  latest: 2023-07-17 EFO 3.56.0
+  latest: 2023-08-15 EFO 3.57.0
   urls:
-    2023-07-17 EFO 3.56.0: https://github.com/EBISPOT/efo/releases/download/v3.56.0/efo.owl
+    2023-08-15 EFO 3.57.0: https://github.com/EBISPOT/efo/releases/download/v3.57.0/efo.owl
 HANCESTRO:
-  latest: 2022-07-18 (2.6)
+  latest: 3.0
   urls:
-    2022-07-18 (2.6): https://github.com/EBISPOT/ancestro/raw/2.6/hancestro.owl
+    3.0: https://github.com/obophenotype/uberon/releases/download/v2023-09-05/uberon.owl
```

wrong ontology, this is the uberon URL

```suggestion
    3.0: https://github.com/EBISPOT/hancestro/raw/3.0/hancestro-base.owl
```

---

Ok this is strange. I definitely made a copy-paste error on the initial fix, as you've correctly noted here, but I think my initial change was actually correct. Or at least I definitely copied the right url per [the ticket description](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-curation/616)

---

The ticket description has the wrong link :)

---

for reference tho, these are all supposed to be directly downloadable permalinks to an ontology file. if its not, the ontology processing GHA will fail because its expecting one